src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -88,7 +92,7 @@
 @Slf4j
 @RequiredArgsConstructor
 public class UserInfoWindowController implements Controller<Node> {
-  private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("d MMM");
+  private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("MM/yyyy");
```

Comment:
I don't mind too much but this would normally be part of internationalization.

Reply:
Not sure how the Chinese write their dates. And we Europeans dont use a /

Reply:
what you mean america isnt the standard of the world?!!?!?! I will see how to make it i18n compliant

Reply:
also what you mean with the enum so will update it